openpaas-james/mailbox/encrypted/pgp-utils/src/test/java/com/linagora/openpaas/gpg/EncrypterTest.java

```diff
@@ -0,0 +1,73 @@
+package com.linagora.openpaas.gpg;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.security.Provider;
+import java.security.Security;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.io.ByteSource;
+
+class EncrypterTest {
+    public static final String INPUT = "ga bou zo meuh\r\nBienvenue chez les shadocks.\r\n";
+
+    @BeforeAll
+    static void setUpAll() throws Exception {
+        String bouncyCastleProviderClassName = "org.bouncycastle.jce.provider.BouncyCastleProvider";
+        Security.addProvider((Provider)Class.forName(bouncyCastleProviderClassName).getDeclaredConstructor().newInstance());
+    }
+
+    @Test
+    void encryptEmptyDataShouldBeSupported() throws Exception {
```

Comment:
What happen if i use a wrong public/private key pair? Or a key that does not conform to PGP format?

Reply:
> What happen if i use a wrong public/private key pair?

The email can't be decrypted.

> Or a key that does not conform to PGP format?

Read key would throw

I'll add the tests